_posts/fr/2018-03-28-codelabs-under-the-hood.md

```diff
@@ -4,7 +4,7 @@ title: Codelabs under the hood
 excerpt: À propos d'Eleven Codelabs
 authors:
     - kelfarsaoui
-lang: en
+lang: fr
 permalink: /codelabs-under-the-hood/
```

Comment:
Tu peux ajouter le `fr/` dans le permalink ? (`/fr/codelabs-under-the-hood/`)

l'URL de l'article ne respecte pas les conventions.

Ce changement n'est pas top point de vue referencement, mais bon ...

Reply:
done :)